apps/files/app.js

```diff
@@ -313,7 +313,7 @@ app.use(cloudcmd({
         // treeroottitle: "Project Space"
         treeroot:               HOME,
         treeroottitle:          "Home Directory",
-        upload_max:             process.env.FILE_UPLOAD_MAX || 10485760000,
+        upload_max:             (process.env.FILE_UPLOAD_MAX <= process.env.NGINX_FILE_UPLOAD_MAX) ? process.env.FILE_UPLOAD_MAX : process.env.NGINX_FILE_UPLOAD_MAX || 10485760000,
```

Comment:
Looking at this, it looks like the original was buggy as well. I would expect anything off of process.env to be a string, but this does comparison with integers. We should probably be converting both `process.env.FILE_UPLOAD_MAX` and `process.env.NGINX_FILE_UPLOAD_MAX` to integers prior to comparing them.